src/media/css/app-list.styl

```diff
@@ -41,9 +41,11 @@ $desktop-tile-padding = 20px 50px;
     margin-top: 30px;
 
     li:not(.loadmore) {
-        border-bottom: 1px solid $filters-border;
         padding-top: 15px;
     }
+    li:not(.loadmore):not(:last-child) {
+        border-bottom: 1px solid $filters-border;
```

Comment:
Probably not necessary after my PR. We can either nuke it here or I'll resolve the conflict.